rollup.config.js

```diff
@@ -47,7 +47,7 @@ export default [
     output: { dir: `dist`, format: 'esm' },
     external,
     plugins: [
-      multiInput(),
+      multiInput.default(),
```

Comment:
just because of https://github.com/alfredosalzillo/rollup-plugin-multi-input/issues/72#issuecomment-1509102009